website/docs/registries/packaging.md

```diff
@@ -0,0 +1,33 @@
+# Packaging your project
+
+When you want to share your package over package registry, it needs to be turned into an archive first.
+The archive will then be uploaded to the registry, where it can be downloaded by other users.
+
+Use the `scarb package` command to create an archive of your package.
+You can read about the package compression algorithm and contents in the [Package tarball](./package-tarball) section.
+Basically when you run the command, Scarb gathers the source code of your package along with metadata files, such as the manifest file, and places them in an archive in `target/package` directory.
+
+If you are in a Git repository, Scarb will first check if the repo state is clean and error out in case of any changes present in the Git working directory.
+To bypass this check, you can use the `--allow-dirty` flag.
+
+The next step is package verification.
+After creating the initial archive, Scarb will attempt to unpack it and compile to check for any corruptions in the packaging process.
+If you want to speed up the packaging process, you can disable this step using the `--no-verify` flag.
+
+> [!WARNING]
+> This is a dangerous operation as it can lead to uploading a corrupted package to the registry.
+> Please use with caution.
+
+After successfully completing the whole process, the `{name}-{version}.tar.zst` archive waits in the `target/package` directory for being uploaded, where both `name` and `version` correspond to the values in `Scarb.toml`.
+
+## Publishing the package
+
+To upload your package, you can use the `scarb publish` command.
+Publishing your package over HTTP is not yet supported, therefore, the only way to upload the package is to use a local registry.
+The command takes the `--index` argument that you can use to pass the local directory path where you want to store the packages.
+
+```shell
+scarb publish --index file:///Users/foo/bar
+```
+
+This is only useful when you are [hosting your own registry](./custom-registry).
```

Comment:
That's a bummer that i learn this only after reading the whole page

Reply:
IIUC this is only related to the current `scarb publish`, not the entire page, but I've tried to adjust this section a bit to include information about limitation in the warning bubble at the top of the section, along with the information about uploading to the official registry: 9b94262f6fac4c3dc587597bd06027de93414c7b

Please TAL